README.md

```diff
@@ -1,7 +1,9 @@
 <!--![shiba love MRI scanner](https://raw.githubusercontent.com/learn-neuroimaging/tutorials-and-resources/master/img/dogeLoveScanner_600x400.gif)-->
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-24-orange.svg?style=flat-square)](#contributors-)
+![Github All Contributors](https://img.shields.io/github/all-contributors/learn-neuroimaging/tutorials-and-resources/master?color=blue) 
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
+[![Build Status](https://travis-ci.org/learn-neuroimaging/tutorials-and-resources.svg?branch=master)](https://travis-ci.org/learn-neuroimaging/tutorials-and-resources)
```

Comment:
```suggestion

[![Build Status](https://travis-ci.org/learn-neuroimaging/tutorials-and-resources.svg?branch=master)](https://travis-ci.org/learn-neuroimaging/tutorials-and-resources)
```

Reply:
Just to not have the badges on the same line. No?

Reply:
Yes, that's always there. I have seen a workaround for this and would update accordingly if it works...